packages/larva-tokens/src/brands/variety.json

```diff
@@ -458,6 +476,87 @@
 			"originalValue": "32px",
 			"name": "PRIMARY_XL_LINE_HEIGHT_MOBILE"
 		},
+		"PRIMARY_XS_FONT_WEIGHT_DESKTOP": {
+			"selector": "lrv-a-font",
+			"status": "alpha",
+			"category": "font-weight",
+			"type": "number",
+			"value": "normal",
+			"originalValue": "700",
```

Comment:
Should this be `normal` or `700`? Not sure the significance of the `700` after it's been changed to `normal`.

Reply:
The originalValue property indicates what the value was previously so it being '700' makes sense (for ones that I've updated). The value property is what's read.